exporter/sentryexporter/sentry_exporter.go

```diff
@@ -0,0 +1,395 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sentryexporter
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/getsentry/sentry-go"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+	"go.opentelemetry.io/collector/translator/conventions"
+)
+
+const (
+	sentryStatusUnknown       = "unknown"
+	otelSentryExporterVersion = "0.0.1"
+	otelSentryExporterName    = "sentry.opentelemetry"
+)
+
+// canonicalCodes maps OpenTelemetry span codes to Sentry's span status.
+// See numeric codes in https://godoc.org/github.com/open-telemetry/opentelemetry-proto/gen/go/trace/v1#Status_StatusCode.
```

Comment:
FYI: https://github.com/open-telemetry/opentelemetry-specification/issues/706#issuecomment-664457764

Reply:
Thanks for the heads up. Do you think I should remove the functionality to read `span.status` then in anticipation of the change?

Reply:
It is fine to remove it later, we may do it whole sale.